### ISSUE TITLE
Why isn't the loss of validation on the same scale as the loss of training ?

### ISSUE THREAD

**Issue by Geo99pro** [lyuwenyu/RT-DETR#355]
**Star RTDETR**
请先在RTDETR主页点击**star**以支持本项目
**Hi Sir @lyuwenyu**

I would like to understand something that is not clear to me.
![image](https://github.com/lyuwenyu/RT-DETR/assets/112230136/baabcb8c-df78-4505-b88c-c8b4a5926a9e)

Even normalizing the validation dataset as you said, I still don't understand why the loss value is not of the same order as the train loss.

I mean for example that with an average train loss = 16.9771, I have an average val loss = 0.911.
![image](https://github.com/lyuwenyu/RT-DETR/assets/112230136/3b12f630-c18e-403c-8601-2a24b312447e)

I don't understand why.

Here's the code I use (before taking the average val loss, I divide by the len(data_loader).
![image](https://github.com/lyuwenyu/RT-DETR/assets/112230136/057859e2-5d94-4889-9035-ceac9d735b80)

Can you, please explain me if i'm doing somthing badly. Thanks

**Comment by lyuwenyu** [comment]
https://github.com/lyuwenyu/RT-DETR/blob/5b628eaa0a2fc25bdafec7e6148d5296b144af85/rtdetr_pytorch/src/solver/det_engine.py#L74-L83

I think you can use the same code as train, and use `metric_logger` to print loss

**Comment by Geo99pro** [comment]
Okay sir @lyuwenyu i'm going to try it as you said and see what gonna happen. 

Thanks for your answer sir !

**Comment by meirashaf** [comment]
hello have you solved the problem? @Geo99pro

**Comment by Geo99pro** [comment]
Hi @meirashaf, not yet, I'm still trying to figure it out.

**Comment by Geo99pro** [comment]
> hello have you solved the problem? @Geo99pro

Hi @meirashaf, not yet, I'm still trying to figure it out.
